.ci/Dockerfile.snmpsim

```diff
@@ -0,0 +1,45 @@
+FROM ubuntu:20.04
+
+ARG PORT
+
+# Install Python and pip
+RUN apt-get update && apt-get -y install \
+    python3 \
+    python3-pip \
+    && rm -rf /var/lib/apt/lists/*
+
+# Install snmpsim
+RUN pip3 install snmpsim
+
+# Create a non-privileged user
+RUN useradd -r -s /bin/false snmpsim
+
+# Create directory for data
+RUN mkdir -p /app/data
+
+# Copy the SNMP data file
+COPY snmpsim/data/self.snmprec /app/data/
+
+# Set ownership of the app directory
+RUN chown -R snmpsim:snmpsim /app
+
+# Set working directory
+WORKDIR /app
+
+# Switch to non-privileged user
+USER snmpsim
+
+# Expose the SNMP port
+EXPOSE $PORT $PORT/udp
+
+# Start the SNMP simulator with the specified configuration
+ENTRYPOINT ["snmpsim-command-responder", \
+    "--data-dir=./data", \
+    "--agent-udpv4-endpoint=0.0.0.0:163", \
```

Comment:
I think we should use the `$PORT` value here, so the argument works as expected.

Reply:
Oops, I was aware but forgot to make this change. fixed with https://github.com/logstash-plugins/logstash-integration-snmp/pull/80/commits/34877170ab9f0f842f8f6ca804f0135053da68cf